docs/LV_client/quick_start/LabVIEW_client_quick_start.md

```diff
@@ -2,22 +2,19 @@
 
 ## Prerequisites 
 
-1. Download LabVIEW 2019 (64 bit) 
+1. Download LabVIEW 2019 (64 bit) *approximately 1 hour to download*
```

Comment:
we need to add a link to get LabVIEW 2019(64) (https://www.ni.com/en-us/support/downloads/software-products/download.labview.html#329483)

Reply:
Fyi they need to register to get the installer